release_notes/Service-Fabric-72CU4.md

```diff
@@ -32,6 +32,22 @@ Microsoft Azure Service Fabric 7.2 Fourth Refresh Release Notes
 
 - Service Fabric 7.2 and higher runtime drops support for .NET core Service Fabric apps running with .NET core 2.2 runtime. .NET core runtime 2.2 is out of support from Dec 2019. Service Fabric runtime will not install .NET core runtime 2.2 as part of its dependency. Customers should upgrade their .NET 2.2 runtime SF apps to the next .NET core LTS version 3.1.
 
+Starting with Service Fabric 7.2. CU4, the logic the runtime uses to select the Cluster certificate has changed. This change applies to both clusters with certificates declared by common name, and clusters with certificates declared by thumbprint, which also have a secondary thumbprint declared. 
+
+Previously, Service Fabric selected among the pool of declared certificates, the time-valid certificate that was the longest living (i.e. with the NotAfter farthest in the future). Starting in this CU, Service Fabric will select, among the pool of time-valid certificates, the newest certificate (i.e. with the NotBefore farthest in the future). In certain configurations, this upgrade will cause a change to the cluster certificate. In general this change should have no impact on cluster health. However, if clients of Service Fabric expect Service Fabric to present the previous certificate, they may need to be updated to expect the new certificate (or any valid cluster certificate).
+
+Example
+| Certificate | Thumbprint | NotBefore | NotAfter (expiry) |
+| - | - | - | - |
+| Primary | AAAA... | 1/1/2020 | 1/1/2023 |
+| Secondary | BBBB... | 6/1/2020 | 6/1/2021 |
+
+In SF versions before 7.2. CU4, SF will select AAAA..., because among the two time-valid certificates, it is the furthest living certificate (1/1/2023). Starting in 7.2. CU4, SF will select BBBB..., because among the two, it is the newer certificate (6/1/2020).
+
+Similarly, for clusters with certificates declared by common name, the selected cluster certificate will change if among the certificates with the declared common name, the longest-living certificate is not also the newest certificate.
+
```

Comment:
Please remove; good content, best suited for the main doc (which we should also update).